WinUIB2CApp/appsettings.json

```diff
@@ -0,0 +1,19 @@
+{
+  {
+    "AzureAdB2C": {
+      "Instance": "[Enter the instance url of your B2C tenant, e.g. https://fabrikamb2c.b2clogin.com]",
+      "Domain": "[Enter the domain of your B2C tenant, e.g. fabrikamb2c.onmicrosoft.com]",
+      "TenantId": "[Enter 'common', or 'organizations' or the Tenant Id (Obtained from the Azure portal. Select 'Endpoints' from the 'App registrations' blade and use the GUID in any of the URLs), e.g. da41245a5-11b3-996c-00a8-4d99re19f292]",
```

Comment:
>[Enter 'common', or 'organizations' or the Tenant Id (Obtained from the Azure portal. Select 'Endpoints' from the 'App registrations' blade and use the GUID in any of the URLs), e.g. da41245a5-11b3-996c-00a8-4d99re19f292] [](http://example.com/codeflow?start=19&length=221)

this does not apply to B2C, change to "Copy Directory (tenant) ID from the Overview tab of the app registration page"